CAP/gap/ConstructiveCategoriesRecord.gi

```diff
@@ -18,7 +18,9 @@ IsEnrichedOverCommutativeRegularSemigroup := Concatenation(
 IsAbCategory := Concatenation( [ 
   "ZeroMorphism", 
   "IsZeroForMorphisms", 
-  "AdditiveInverseForMorphisms" ], ~.IsEnrichedOverCommutativeRegularSemigroup ),
+  "SubtractionForMorphisms",
+  "AdditiveInverseForMorphisms",
+  "MultiplyWithElementOfCommutativeRingForMorphisms" ], ~.IsEnrichedOverCommutativeRegularSemigroup ),
```

Comment:
I added `"MultiplyWithElementOfCommutativeRingForMorphisms"` to this list, as I found `"AdditiveInverseForMorphisms"` at the end of it, and commutative regular semigroups do not have additive inverses in the strict sense. Maybe I should create a new list with a name like `IsEnrichedOverRlinearCommutativeRegularSemigroup`.